system/modules/core/library/Contao/System.php

```diff
@@ -692,9 +692,37 @@ protected static function convertXlfToPhp($strName, $strLanguage, $blnLoad=false
 
 		// Use loadXML() instead of load() (see 7192)
 		$xml->loadXML(file_get_contents(TL_ROOT . '/' . $strName));
-
 		$return = "\n// $strName\n";
-		$units = $xml->getElementsByTagName('trans-unit');
+
+		$objFileNodes     = $xml->getElementsByTagName('file');
+		$strCheckLanguage = strtolower($strLanguage);
```

Comment:
The case-insensitive comparison is part of the contao/core-bundle#654 bugfix, because the XLIFF standard defines that languages are case-insensitive.